controllers/secret.go

```diff
@@ -24,187 +24,108 @@ import (
 	certv1alpha1 "github.com/redhat-openshift-ecosystem/operator-certification-operator/api/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 	k8errors "k8s.io/apimachinery/pkg/api/errors"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 const (
-	KUBECONFIG_SECRET = "kubeconfig"
-	GITHUB_API_SECRET = "github-api-token"
-	GITHUB_TOKEN      = "GITHUB_TOKEN"
-	PYXIS_API_SECRET  = "pyxis-api-secret"
-	PYXIS_API_KEY     = "pyxis_api_key"
+	KUBECONFIG_SECRET_NAME     = "kubeconfig"
+	KUBECONFIG_SECRET_KEY_NAME = "kubeconfig"
+	GITHUB_API_SECRET_NAME     = "github-api-token"
+	GITHUB_API_KEY_NAME        = "GITHUB_TOKEN"
+	PYXIS_API_SECRET_NAME      = "pyxis-api-secret"
+	PYXIS_API_KEY_NAME         = "pyxis_api_key"
```

Comment:
Can we either remove `secret` from the kube `key` or add `secret` to all the other `keys` so we are consistant?

Reply:
yes can do.